pkg/agent/openflow/client.go

```diff
@@ -312,6 +315,43 @@ func (c *client) addFlows(cache *flowCategoryCache, flowCacheKey string, flows [
 	return nil
 }
 
+func (c *client) hardModifyFlows(cache *flowCategoryCache, flowCacheKey string, flows []binding.Flow) error {
```

Comment:
Do we already have a "soft" modifyFlows? If not, I think naming it "modFlows" is just fine.

Reply:
Updated.